cdktf-provider-sync

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 <destination-directory>"
+    exit 1
+fi
+
+if [ ! -f "cdktf.json" ]; then
+    echo "cdktf.json does not exist"
+    exit 1
+fi
+
+cdktf get --parallelism 1 --output "$1"
+
+for i in "$1"/*; do
+    if [ ! -d "$i" ]; then
+        continue
+    fi
+    dest=$1/cdktf_cdktf_provider_$(basename "$i")
+    echo "Moving $i to $dest"
```

Comment:
why `mv` ?

Reply:
`cdktf get` creates directories based on the provider names, e.g., for the random provider, it's just `random`. The pre-built cdktf random Python module uses `cdktf_cdktf_provider_random` as the name. By renaming the directories, we can keep our current Python imports in the stack files and be compatible with the pre-built packages.